main.tf

```
@@ -123,12 +123,14 @@ module "ssm" {
 
 module "webhook" {
   source = "./modules/webhook"
-
+  ssm_paths = {
+    root = "${local.ssm_root_path}"
```

Root module was not working
```suggestion
    root = "${local.ssm_root_path}"
    webhook = "${var.ssm_paths.webhook}"
```